repository-data/application/src/main/resources/hcm-config/configuration/update/Migration20180313UpdateGeographicCoverage.groovy

```diff
@@ -0,0 +1,106 @@
+package org.hippoecm.frontend.plugins.cms.admin.updater
+
+import org.onehippo.forge.content.exim.core.DocumentManager
+import org.onehippo.forge.content.exim.core.impl.WorkflowDocumentManagerImpl
+import org.onehippo.repository.update.BaseNodeUpdateVisitor
+import org.apache.jackrabbit.value.StringValue
+import javax.jcr.*
+
+class Migration20180313UpdateGeographicCoverage extends BaseNodeUpdateVisitor {
+
+    private static final String GEOGRAPHIC_COVERAGE_PROPERTY = "publicationsystem:GeographicCoverage"
+
+    List typesWithGeographicCoverage = [
+        "publicationsystem:publication",
+        "publicationsystem:dataset"
```

Comment:
Do you need to update indicators/legacy as well?

Reply:
For legacy, they haven't got not any real data in yet, but Dave is happy for us to update the doc type so that they are consistent, and then look to update the migration to map to the new values.  Will update this groovy script to include both indicators and legacy (currently shouldn't be any real values , but if there is this will catch them).